### PR TITLE
[ARB-34]-Endpoint to get Animal Record

### PR DESCRIPTION
**Pull request by enavarroAK**
### Changelog
- Created endpoint to get the animal record
- Created the animal record type.

### Ticket 
https://github.com/akurey/animal-rescue-backend/issues/34

### Solution
- Created the endpoint to get one animal record.

### Considerations
The endpoint uses AFN_GetAnimalReport since it was easier to get the returning data.

Example response:
`{
    "response": [
        {
            "id": 2,
            "IdAnimal": 0,
            "AnimalName": "",
            "ScientificName": "",
            "ConservationStatusName": "",
            "Abbreviaton": "NT",
            "ClassificationName": "",
            "Fields": "{\"Sexo\": \"Hembra\", \"Color\": \"Amarillo\", \"Dirección\": \"{\\\"Canton\\\": \\\"Osa\\\", \\\"Distrito\\\": \\\"Bahia Drake\\\", \\\"Direccion\\\": \\\"PN Corcovado\\\", \\\"Provincia\\\": \\\"Puntarenas\\\"}\", \"Procedencia\": \"Osa\", \"Número de identificación (CHIP)\": \"67890\"}"
        }
    ]
}`

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,7 +1,7 @@
-DB_HOST="ec2-52-3-60-53.compute-1.amazonaws.com"
+DB_HOST="otto.db.elephantsql.com"
```
Comment (rnavarroSF) [inline .env:1]:
```suggestion
DB_HOST="ec2-52-3-60-53.compute-1.amazonaws.com"
```

File: .env [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 DB_PORT=5432
-DB_USER="xybvtiinusasyh"
-DB_PASSWORD="389e9dac39c30290ed036e62362ec3e7cc435f0432f3b2cdce1bf6125bb61ae3"
-DB_NAME="d63vklqd241lbk"
+DB_USER="rutelazj"
+DB_PASSWORD="ZN1uXyCCGjXbIs5KR-3NlLSJjVrPFVTT"
+DB_NAME="rutelazj"
```
Comment (rnavarroSF) [inline .env:5]:
```suggestion
DB_USER="xybvtiinusasyh"
DB_PASSWORD="389e9dac39c30290ed036e62362ec3e7cc435f0432f3b2cdce1bf6125bb61ae3"
DB_NAME="d63vklqd241lbk"
```

Lets revert this until dicember

File: controllers/report.go [hunk 3]
```diff
@@ -11,6 +11,17 @@ import (
 
 type ReportController struct{}
 
+func (ctrl ReportController) GetAnimalRecord(context *gin.Context) {
+	var report []*models.AnimalReport
```
Comment (rnavarroSF) [inline controllers/report.go:15]: Doesn't need to be an array, we will only be getting one record

**Review by rnavarroSF — CHANGES_REQUESTED** [review]
Couple of comments

**Review by pdinarte — APPROVED** [review]
🚀

**Review by rnavarroSF — APPROVED** [review]
Thanks so much for everything! 🚀

**Pull request merged**